table/queries/derived_satellite_scans.sql

```diff
@@ -153,6 +153,17 @@ AS (
 );
 
 
+# Get all domains that have even a single valid HTTPS certificate resolution per scan
+CREATE OR REPLACE TABLE `PROJECT_NAME.DERIVED_DATASET.https_capable_domains`
```

Comment:
Add a TODO to do this in the pipeline, so it's cheaper and incremental.

Reply:
done